android/app/src/main/java/com/zulipmobile/notifications/FcmListenerService.kt

```diff
@@ -1,12 +1,10 @@
 package com.zulipmobile.notifications
 
-import com.zulipmobile.notifications.onReceived
 import com.google.firebase.messaging.FirebaseMessagingService
 import com.google.firebase.messaging.RemoteMessage
 import com.zulipmobile.MainApplication
 import com.facebook.react.bridge.ReactContext
 import com.facebook.react.ReactApplication
-import com.zulipmobile.notifications.NotificationsModule
 
 class FcmListenerService : FirebaseMessagingService() {
     override fun onMessageReceived(message: RemoteMessage) {
```

Comment:
```
android [nfc]: Remove some auto-generated imports that are redundant

The Java-to-Kotlin conversion added these import lines, but they
aren't necessary because there's already a line

  package com.zulipmobile.notifications

at the top.
```

Hmm, I think I read too quickly: I thought that line `package com.zulipmobile.notifications` was

```kotlin
import com.zulipmobile.notifications
```

so I'm not sure why it's right to remove the two lines touched in this commit. I was getting an "Unused import directive" lint on them; maybe the commit message could say just that.

Reply:
That's the same package as for NotificationUiManager.kt. The consequence is that declarations in the one file can be freely used in the other, without the need for an import statement.

This isn't particularly clear in the docs:
https://kotlinlang.org/docs/packages.html

but happily it is clear in the spec:
https://kotlinlang.org/spec/packages-and-imports.html#importing